vendor/bat-native-ads/src/bat/ads/internal/ad_targeting/geographic/subdivision/get_subdivision_url_request_builder.cc

```diff
@@ -29,7 +29,7 @@ UrlRequestPtr GetSubdivisionUrlRequestBuilder::Build() {
 ///////////////////////////////////////////////////////////////////////////////
 
 std::string GetSubdivisionUrlRequestBuilder::BuildUrl() const {
-  return base::StringPrintf("%s/v5/getstate", server::GetHost().c_str());
+  return base::StringPrintf("%s/v1/getstate", serve::server::GetHost().c_str());
```

Comment:
@tmancey ticket states "Expected result: v5/getstate end-point should success and provide the regions subdivision (United States)", are we intending to serve from `v1`?

Reply:
@moritzhaller Please see https://github.com/brave/brave-core/pull/7453#discussion_r544127704, thanks

Reply:
Ah yes, good spot sir